x-pack/solutions/observability/plugins/apm/public/components/shared/summary/build_url.ts

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
This is not a react component, could we move it somewhere else? And maybe not withing a **summary** folder, since this could be used anywhere not only in the summary/

Reply:
moved to `utils` 👍